module-types/Datacom/PSU-200-DC-B.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 200 DC-B
+part_number: 800.5308.xx
+comments: DC power supply with 48V-60Vdc input. Forced ventilation with airflow entering through the front panel of the power supply. Allows operation as
+  a redundant.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be